docs/releases/v1.0.0.rst

```diff
@@ -179,6 +179,7 @@
     -[api] Method :meth:`dt.FExpr.re_match` has been deprecated and replaced
       with a function :func:`dt.re.match()`. [#3017]
 
+    -[enh] Added reducer function :func:`countna` [#2999]
```

Comment:
This would have to be moved to file `v1.1.0.rst`.

Reply:
(You'll need to merge `main` into this branch)

Reply:
Ack